contracts/contracts/sbtc-registry.clar

```diff
@@ -44,10 +48,16 @@
   }
 )
 
-;; Read-only functions
+;; Data structure to store aggregate pubkey,
+;; stored to avoid replay
+(define-map aggregate-pubkeys (buff 32) bool)
 
+;; Data structure to store the current signer set,
+;; stored to avoid replay
+(define-map multi-sig-address principal bool)
```

Comment:
I think my recollection was that we'd only want to prevent aggregate key replays? Although I don't really think this is an issue. The only downside is that this says "you can't rotate back to an older set of keys", which seems technically valid, but maybe that's more of a good thing to prevent it.

Reply:
Right. I do remember confirming only agg-key but when writing it out I thought the multi-sig made sense as well.